pod/pkg/metrics/metrics.go

```diff
@@ -76,7 +79,15 @@ func init() {
 		Help:      "errors from pod perceiver operations",
 	}, []string{"stage", "errorName"})
 
+	podsAnnotated = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Will this treat every pod separately?  So if there's 1000 pods in the system, there will be 1000 separate counts maintained?

Just curious if @jayunit100 could chime in and let us know if that's something that prometheus supports -- I thought I read something that said they try to minimize the number of independent metrics for performance or to avoid blowing up storage.

Reply:
If I understand the metrics correctly this will create a metric for each pod in the system.  Same thing for images too.  My thought here is if a pod/image is continually being updated then it might be worth looking at. In theory these counters should be very close to 1.

We could move to a single counter for all annotations, but I'm not sure that is as useful.